### ISSUE TITLE
Seems like @Collection() accessor not working

### ISSUE THREAD

**Issue by SSebigo** [isar/isar#179]
**Steps to Reproduce**
My understanding of ```@Collection``` accessor param is that it is used to generate a "getter" with the given name.

For example, if I have a class ```IsarUser``` without accessor I'll have to use ```isar.isarUsers``` to access the collection. But if I give the accessor name ```users``` (```@Collection(accessor: "users")```) then I can use ```isar.users```.

The problem is that when given an accessor name, the generator seems to completely ignore it.

It should generate something like this:
```dart
extension GetIsarUserCollection on Isar {
  IsarCollection<IsarUser> get users {
    return getCollection('IsarUser');
  }
}
```
but instead, it still generates something like that:
```dart
extension GetIsarUserCollection on Isar {
  IsarCollection<IsarUser> get isarUsers {
    return getCollection('IsarUser');
  }
}
```

**Code sample**
```dart
@Collection(accessor: 'users')
class IsarUser {
  IsarUser({
    required this.email,
    this.id,
  });

  final String email;
  final int? id;
}
```

**Version**
 - Platform: iOS, Android, Mac, Windows, Linux, Web
 - Flutter version: [2.8.1]
 - Isar version: [1.0.5]
 - Isar Generator version: [1.0.5]

**Comment by LeonFuss** [comment]
Im not sure what accessor is used for, but to change the name just use the @Name annotation :)
`@Collection()
@Name(“User“)
class IsarUser ({…})`

**Comment by leisim** [comment]
@SSebigo thanks for reporting, that seems to be a bug. will be resolved in the next release :)

**Comment by SSebigo** [comment]
@LeonFuss @Name("User") won't work for this use case, it doesn't generate the getter for the collection.

**Comment by LeonFuss** [comment]
Good to know :)

**Comment by leisim** [comment]
Fixed in `2.0.0-dev.0`
